rest-client/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>sdk-group</artifactId>
         <groupId>org.sagebionetworks.bridge</groupId>
-        <version>0.13.11</version>
+        <version>0.13.12</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
```

Comment:
Change /v3/studies/:identifier to indicate that the user is not an admin or worker (rather than just not an admin), to document this is available. Also add to API classes that have the method (_For Workers, I think).

Reply:
Fixed.